schemas/fb-linux.yml

```diff
@@ -0,0 +1,56 @@
+---
+- src: "{app_name}_{version}_debian-{os_version}_{arch}.deb"
+  arch:
+    - amd64
+    - arm64
+  uploads:
+    - type: apt
+      src_repo: "http://download.newrelic.com/infrastructure_agent/linux/apt"
+      dest: "{dest_prefix}linux/apt/"
+      os_version:
+        - buster
+        - jessie
+        - stretch
+
+- src: "{app_name}_{version}_ubuntu-{os_version}_{arch}.deb"
+  arch:
+    - amd64
+  uploads:
+    - type: apt
+      src_repo: "http://download.newrelic.com/infrastructure_agent/linux/apt"
+      dest: "{dest_prefix}linux/apt/"
+      os_version:
+        - xenial
+
+- src: "{app_name}_{version}_ubuntu-{os_version}_{arch}.deb"
+  arch:
+    - amd64
+    - arm64
+  uploads:
+    - type: apt
+      src_repo: "http://download.newrelic.com/infrastructure_agent/linux/apt"
+      dest: "{dest_prefix}linux/apt/"
+      os_version:
+        - bionic
+        - focal
+
+- src: "{app_name}-{version}-1.centos-{os_version}.{arch}.rpm"
```

Comment:
Just curious to know why `-1` after version 🤔 
Maybe a comment will be helpful also. WDYT?

Reply:
I had to search. Loolks like it's the `[-debian-revision]`
https://askubuntu.com/questions/441879/why-do-some-packages-have-extra-numbers-before-a-colon-on-the-front-of-their-ver